src/ape/api/accounts.py

```diff
@@ -111,6 +130,9 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
 
     @cached_property
     def _convert(self) -> Callable:
+        """
+        Converts the account.
```

Comment:
You don't need to document this property.

it is considered "protected" and should not be called by users in a supported fashion.

You can tell because it is prefixed with a singel `_`.